README.md

```diff
@@ -74,6 +74,49 @@ If you use another CPU and would like to make sure we test it or discuss adding
 an assembly optimized algorithm implementation, please open an issue to discuss
 adding it to our CI.
 
+## Platform Support
+
+AWS-LC builds and runs on a variety of CPU architectures. The following is an overview of the 
+platforms we actively support or are known to be of interest to our community. Regardless of 
+our support level for a particular platform, we will gladly consider contributions that can 
+improve or extend our support for it.
+
+### Supported Platforms
+
+The following platforms are actively tested in our CI pipeline. For each pull request, the 
+proposed change is validated to confirm that it successfully builds and tests pass for these 
+platform. A more complete description of our test setup can be found in the 
+[CI README](https://github.com/aws/aws-lc/blob/main/tests/ci/README.md).
```

Comment:
Should this be combined with `Compiler, OS, and CPU support` up above, it seems very related.